frontend/src/components/Register.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect } from 'react'
+import { useSelector, useDispatch, batch} from 'react-redux'
+import { useHistory, Link } from 'react-router-dom'
+
+import user from '../reducers/user'
+
+import { API_URL } from '../reuseables/urls'
+import { MainContainer, Header, Form, Label, InputField, Text, SecondaryButtonContainer, PrimaryButton, SecondaryButton, ErrorMessage, InputWrapper } from './styled-components/form-style'
```

Comment:
when you import more than 3 items from the same library, then organize them into a column. it is easier to read and maintain.